portal-api/src/v1/controllers/utilisation-report-service/utilisation-report-upload.controller.js

```diff
@@ -1,8 +1,22 @@
 const sendEmail = require('../../email');
+const api = require('../../api');
 const { EMAIL_TEMPLATE_IDS } = require('../../../constants');
 
 const { PDC_INPUTTERS_EMAIL_RECIPIENT } = process.env;
 
+const getPaymentOfficerTeamDetailsFromBank = async (bankId) => {
+  const { data } = await api.getPaymentOfficerTeamDetailsFromBank(bankId);
```

Comment:
this will throw if it returns an error, can we either wrap in try/catch or do object destructuring later doing void checks as Abhi has suggested below please?